docassemble/AssemblyLine/al_document.py

```diff
@@ -1419,18 +1426,18 @@ def has_overflow(self):
     """
     return False
   
-  def as_list(self, **kwargs) -> List[DAFile]:
-    return [self]
+  def as_list(self, key:str='final', refresh:bool=True, **kwargs) -> List[DAFile]:
+    return [self[key]]
```

Comment:
TBH I don't know if I understand how this class is supposed to work, and there's no example usage that I can find.

But this defaults to `ALDocument.__getitem__`, which defaults to `DADict.__getitem__`, which is returning a different object than `self`, right?

Reply:
Looking closer i think you're right, this may have been a bug in code that just hasn't been used yet. The idea of this class is to let you just use any DAFile as an ALDocument and not have the bundle change it in any of the places it is rendered.